Tests/FluentAssertions.Specs/Formatting/PredicateLambdaExpressionValueFormatterSpecs.cs

```diff
@@ -11,6 +11,38 @@ public class PredicateLambdaExpressionValueFormatterSpecs
 {
     private readonly PredicateLambdaExpressionValueFormatter formatter = new();
 
+    [Fact]
+    public void Constructor_expression_with_argument_is_not_reduced()
```

Comment:
🤔 "Reduced"? I didn't get it.

Reply:
Yeah... I also struggled with naming this one.

`ConstantSubExpressionReductionVisitor` tries to "reduce" expressions, but threw an exception when trying to reduce this one.

Brain storming better names:
`Constructor_expression_with_argument_can_be_formatted`
`Formatting_a_constructor_expression_with_argument_does_not_throw`

Reply:
> Constructor_expression_with_argument_can_be_formatted

👍